### PR TITLE
Additions to beacons module

### PR DESCRIPTION
**Pull request by garethgreenaway**
adding code to allow enabling and disabling specific beacons. #23400

### CODE REVIEW

File: salt/modules/beacons.py [hunk 1]
```diff
@@ -251,3 +251,93 @@ def disable(**kwargs):
             # Effectively a no-op, since we can't really return without an event system
             ret['comment'] = 'Event module not available. Beacons enable job failed.'
     return ret
+
+
+def enable_beacon(name, **kwargs):
+    '''
```
Comment (rallytime) [inline salt/modules/beacons.py]: Can you add some version added directives to these new functions?
```
.. versionadded:: Beryllium
```

Comment (garethgreenaway) [inline salt/modules/beacons.py]: I can.  They were all added new for Beryllium though and there is a verionadded at the top of the module, should I add the versionadded for all of them?

Comment (rallytime) [inline salt/modules/beacons.py]: Hrm...I guess I thought beacons were backported to 2015.5. I'll check that and change it if necessary. Nothing to do here, then. :)

Thanks!

Comment (rallytime) [inline salt/modules/beacons.py]: Oh, beacons/__initi__.py was added in 2015.5, but not modules/beacons.py. Sorry about the confusion there!

Comment (garethgreenaway) [inline salt/modules/beacons.py]: No worries.

**Pull request merged**